.github/workflows/ci.yml

```diff
@@ -47,6 +47,8 @@ jobs:
         pip install opencv-python-headless
         # install parking-env to test HER (pinned so it works with gym 0.21)
         pip install highway-env==1.5.0
+        # Install a test env to test gym-packages arg
```

Comment:
maybe add those instructions somewhere?

Reply:
about how to install the test package?

Reply:
yes because without that you cannot run the tests locally, no?

Reply:
True! Where would you add the instructions?

Reply:
Either readme or in the doc (there should be a section on testing the package, if not, you can create one)